apps/heureka/README.md

```diff
@@ -73,7 +73,11 @@ We use [MockServiceWorker](https://mswjs.io/) (MSW) to intercept GraphQL request
    ```bash
    cd apps/heureka
    ```
-2. Update GraphQL types from the latest GraphQL schema:
+2. Create `.env` file if you haven't already and change configurations accordingly:
+   ```bash
+   cp .env.template .env
```

Comment:
This .env.template file lives wihin the application or?
would it make sense to mention the path  where to find the `app/heureka/.env.template` and where to copy it?

Reply:
In the step `1.` user is already in `heureka` app directory where running `cp .env.template .env` will create the file.

Reply:
Completely right! the step 1 wasn't in the diff displayed.